### PR TITLE
[MM-61512] Fix errcheck issues in server/channels/app/users/helper_test.go

### PR DESCRIPTION
**Pull request by Honsei901**
#### Summary

Fix errcheck issues in server/channels/app/users/helper_test.go

#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/29130#
Jira [MM-61512](https://mattermost.atlassian.net/browse/MM-61512)

#### Release Note

```release-note
NONE
```

[MM-61512]: https://mattermost.atlassian.net/browse/MM-61512?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Honsei901** [comment]
Hi @hanzei, Can I see the error details in Enterprise CI/tests to fix?

File: server/channels/app/users/helper_test.go [hunk 1]
```diff
@@ -69,7 +70,9 @@ func setupTestHelper(s store.Store, includeCacheLayer bool, tb testing.TB) *Test
 	*config.PasswordSettings.Uppercase = false
 	*config.PasswordSettings.Symbol = false
 	*config.PasswordSettings.Number = false
-	configStore.Set(config)
+
+	_, _, err = configStore.Set(config)
```
Comment (hanzei) [inline server/channels/app/users/helper_test.go:74]: Nit:
```suggestion
	_, _, err = configStore.Set(config)
```

**Review by hanzei — APPROVED** [review]
LGTM

File: server/channels/app/users/helper_test.go [hunk 2]
```diff
@@ -69,7 +70,9 @@ func setupTestHelper(s store.Store, includeCacheLayer bool, tb testing.TB) *Test
...
+	require.NoError(tb, err, "Failed to set config")
```
Comment (agnivade) [inline server/channels/app/users/helper_test.go:75]: Just `require.NoError(tb, err)` is sufficient.

Comment (Honsei901) [reply server/channels/app/users/helper_test.go]: I've fixed the issue.

**Review by agnivade — APPROVED** [review]

**Pull request merged**